### PR TITLE
Fix cluster apis

### PR DESCRIPTION
**Pull request by delvedor**
As titled.

`cluster.reroute` and `cluster.state`, according to the ES team, should not be typed, as they return the internal cluster state which isn't considered reliable.

### CODE REVIEW

File: specification/cluster/reroute/ClusterRerouteResponse.ts [hunk 1]
```diff
@@ -17,12 +17,13 @@
  * under the License.
  */
 
-import { AcknowledgedResponseBase } from '@_types/Base'
-import { RerouteExplanation, RerouteState } from './types'
+import { UserDefinedValue } from '@spec_utils/UserDefinedValue'
 
-export class Response extends AcknowledgedResponseBase {
-  body: {
-    explanations?: RerouteExplanation[]
```
Comment (Mpdreamz) [inline specification/cluster/reroute/ClusterRerouteResponse.ts:25]: Explanations should be stable? Understood stood state is untyped.

**Review by philkra — APPROVED** [review]
LGTM

File: specification/cluster/reroute/ClusterRerouteResponse.ts (hunk 1 above)
Comment (delvedor) [reply specification/cluster/reroute/ClusterRerouteResponse.ts:25]: According to the es-distrib team, both this APIs don't have any stability guarantee.

Comment (Mpdreamz) [reply specification/cluster/reroute/ClusterRerouteResponse.ts:25]: kk I would challenge the `explanations` portion of this API. Now that we have a spec we can say this API is stable but the cluster state it exposes is just a map/object/dict/whatever with no guarantees.

**Pull request merged**